src/charm.py

```diff
@@ -799,18 +803,42 @@ def _configure_bessd_for_dpdk(self) -> None:
         if not dpdk.is_configured(container_name=self._bessd_container_name):
             dpdk.configure(container_name=self._bessd_container_name)
 
+    def _are_routes_created(self) -> bool:
```

Comment:
Nit but all the other validators are written in the form `x_is_done` or whatever, not in the question form like here (`is_x_done`). Let's follow the same approach here and use `_routes_are_created` for readability.